engine/src/io/DataLoader.cpp

```diff
@@ -150,6 +150,10 @@ std::unique_ptr<ral::frame::BlazingTable> data_loader::get_metadata(int offset)
 	if (metadata_batches.size() == 1){
 		return std::move(metadata_batches[0]);
 	} else {
+		if( ral::utilities::checkIfConcatenatingStringsWillOverflow(metadata_batche_views)) {
+			std::cout<<"WARNING: In data_loader::get_metadata Concatenating Strings will overflow strings length"<<std::endl;
```

Comment:
Have this std::cout be output to the logs.
You wont have a query id or anything like that, so you can leave all the logging fields blank, except for the info where you provide this warning.